features/ink.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-09-09
-name: Ink API
-description: TODO
+name: Ink
+description: The `navigator.ink` API draws between pointer events, reducing the delay between moving a stylus or cursor and a pen stroke appearing on screen.
```

Comment:
I think "draws between pointer events" is not entirely correct. I think it's more something like "draw before pointer events are dispatched to the webpage". How about:

```suggestion
description: The `navigator.ink` API uses the system compositor to draw to a `<canvas>` element ahead of pointer events, reducing the delay between moving a stylus or cursor and a pen stroke appearing on screen.
```

Reply:
I like this. It makes more sense this way. 👍 I see now that my version implied that drawing only happens with events, which is also wrong.